docs/package.json

```diff
@@ -23,10 +23,10 @@
     "prod:deploy": "node scripts/deploy/aws.js",
     "prettier-all": "npx prettier --write '{src,static-config-helpers,static-config-parts,static.config,scripts,lib}/**/*.{js,jsx}'",
     "preversion": "yarn run lint && yarn run build",
-    "format": "yarn run lint --fix && prettier-all",
+    "format": "yarn run lint --fix && yarn run prettier-all",
     "lint": "eslint 'src/**/*.js' 'static-config-*/*.js' 'src/**/*.js' 'static.config.js'",
     "check-ci": "yarn run lint",
-    "test-local": "node test/spectacle-integration.js"
+    "test-local": "yarn run start && node test/spectacle-integration.js"
```

Comment:
This isn't going to work because `yarn start` never ends -- it stays a running process waiting on HMR changes. We need to document the user needs to run `yarn start` in a different terminal, then run `yarn test-local`